ouroboros-consensus/test/consensus-test/Test/Consensus/MiniProtocol/LocalStateQuery/Server.hs

```diff
@@ -144,7 +148,7 @@ checkOutcome k chain = conjoin . map (uncurry checkResult)
         | otherwise
         -> tabulate "Acquired" ["AcquireFailurePointNotOnChain"] $ property True
       Left AcquireFailurePointTooOld
-        | pointSlot pt >= immutableSlot
+        | pointSlot pt >= immutableSlot   -- TODO what if the immtip is a multi-leader slot?
```

Comment:
Then we ask for a point with the same slot as the immutable tip, but a different hash, we get a `AcquireFailurePointNotOnChain`, right?
https://github.com/IntersectMBO/ouroboros-consensus/blob/ef8186daf6bd70fc82d7df5817cf033f44a0ee6f/ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/MiniProtocol/LocalStateQuery/Server.hs#L47-L51